updatecli/updatecli.d/jenkins-lts.yaml

```diff
@@ -0,0 +1,52 @@
+---
+# This pipeline is used to bump the recommended Jenkins core version in the properties file.
+name: Bump the recommended Jenkins core version in the versions.properties file
+# SCM configuration for GitHub repository
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+# Source configuration to get the previous Jenkins LTS version
+sources:
+  JenkinsPreviousLTS:
+    name: Get the previous Jenkins LTS version
+    kind: shell
+    spec:
+      command: bash ./updatecli/scripts/jenkins-lts.sh 1 # Shell command to retrieve the previous Jenkins LTS version
+
+# Condition to test that the previous LTS Jenkins version exists
+conditions:
+  jenkinsPreviousLTSVersion:
+    kind: jenkins
+    sourceid: JenkinsPreviousLTS
+
+# Target configuration to update the Jenkins LTS version in the versions.properties file
+targets:
+  update-jenkins-lts-version:
+    name: "Update Jenkins LTS version in versions.properties"
+    kind: file
+    spec:
+      file: ./plugin-modernizer-core/src/main/resources/versions.properties # Path to the versions.properties file
```

Comment:
Maybe we will need to store the version on the pom.xml and use maven filtering to have this value on the recipes.yml

Or we need to "hardcode" this on java recipe.

I'm not sure yet what's the best